From cbcfdf5c48a38330beb42fb30a0318cc86fa5791 Mon Sep 17 00:00:00 2001
From: Gary Zhang <b13634@freescale.com>
Date: Tue, 28 Aug 2012 17:15:42 +0800
Subject: [PATCH 1976/2463] ENGR00221689 wm8962: add wait time after enable
 power supply

add 100ms wait time after enable power supply for
power stability

Signed-off-by: Gary Zhang <b13634@freescale.com>
---
 sound/soc/codecs/wm8962.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/sound/soc/codecs/wm8962.c b/sound/soc/codecs/wm8962.c
index 8558be5..b145311 100644
--- a/sound/soc/codecs/wm8962.c
+++ b/sound/soc/codecs/wm8962.c
@@ -5,6 +5,7 @@
  *
  * Author: Mark Brown <broonie@opensource.wolfsonmicro.com>
  *
+ * Copyright (C) 2012 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License version 2 as
@@ -4019,7 +4020,7 @@ static int wm8962_probe(struct snd_soc_codec *codec)
 		dev_err(codec->dev, "Failed to enable supplies: %d\n", ret);
 		goto err_get;
 	}
-
+	msleep(100);
 	ret = snd_soc_read(codec, WM8962_SOFTWARE_RESET);
 	if (ret < 0) {
 		dev_err(codec->dev, "Failed to read ID register\n");
-- 
1.7.7.4

